wordpress-mobile/WordPress-iOS
# NotificationContentRange: Class moved to Notification folder

Fixes parts of #9758 

This PR was created on top of #9861, and organized some Notification related code inside the Notification folder.

I also changed the `FormattableCommentRange` to `NotificationCommentRange`, since it's a subclass of `NotificationContentRange` and is Notification specific.

To test:
- Build and run the project.
- Check that the notification section works as it should.
- Check that the Activity Log section works as it should.
